### PR TITLE
feat: Added Hungarian language

### PR DESCRIPTION
**Pull request by zalnaRs**
## Description
Added Hungarian language

### CODE REVIEW

File: src/shared/locales/all.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ import eleLocaleVi from 'element-ui/lib/locale/lang/vi'
 import eleLocaleZhCN from 'element-ui/lib/locale/lang/zh-CN'
 import eleLocaleZhTW from 'element-ui/lib/locale/lang/zh-TW'
 import eleLocaleUk from 'element-ui/lib/locale/lang/ua'
+import eleLocaleHu from 'element-ui/lib/locale/lang/hu'
```
Comment (agalwood) [inline src/shared/locales/all.js:19]: Please let import also be in alphabetical order.

File: src/shared/locales/all.js [hunk 2]
```diff
@@ -34,6 +35,7 @@ import appLocaleVi from '@shared/locales/vi'
 import appLocaleZhCN from '@shared/locales/zh-CN'
 import appLocaleZhTW from '@shared/locales/zh-TW'
 import appLocaleUk from '@shared/locales/uk'
+import appLocaleHu from '@shared/locales/hu'
```
Comment (agalwood) [inline src/shared/locales/all.js:38]: Please let import also be in alphabetical order.

Comment (zalnaRs) [reply src/shared/locales/all.js]: fixed

File: src/shared/locales/all.js (hunk 1 above)
Comment (zalnaRs) [reply src/shared/locales/all.js]: fixed

**Comment by agalwood** [comment]
Tks

**Pull request merged**